.github/workflows/patch_build.yml

```diff
@@ -5,6 +5,9 @@ on:
   push:
     branches:
       - patch
+env:
+  ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+  RAILS_MASTER_KEY: ${{ secrets.RAILS_MASTER_KEY }}
```

Comment:
the casing of from secrets comes from what we call them on github when defining them, this will just break

Reply:
I see now that this is how they are defined, so....i guess they never worked before.